guides/doc-Monitoring_Project/topics/proc_enabling-access-to-metrics-via-web-ui.adoc

```diff
@@ -0,0 +1,26 @@
+[id='enabling-access-to-telemery-data-via-web-ui_{context}']
+= Enabling Access to Metrics via the Web UI
+
+This procedure describes how to access metrics collected by PCP, via the web UI.
+
+.Procedure
```

Comment:
This procedure is incomplete, as I didn't want to copy paste too much stuff yet.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/monitoring_and_managing_system_status_and_performance/setting-up-graphical-representation-of-pcp-metrics_monitoring-and-managing-system-status-and-performance#doc-wrapper describes the setup

Reply:
We can also put this link `For more information` to redirect.